day_1/questions.md

```diff
@@ -1,13 +1,20 @@
 ## Day 1 Questions
 
 1. How would you log the string `"Hello World!"` to the console?
+console.log("Hello World!")
 
 1. What is the character you would use to indicate comments in a JavaScript file?
+// comments would go here
 
 1. Explain the difference between an integer and a floating-point number?
+Integer is a positive whole number (2, 3, 4). While a floating-point is a a negative number or a non-whole number (-5, 1.3, 0.22).
```

Comment:
Just a note here, both integers and floats can be _positive_ or _negative_.  -23 or 5 are both integers. -4.78 or 28.987 are both floats.